rakam-spi/src/test/java/org/rakam/analysis/TestFunnelQueryExecutor.java

```diff
@@ -199,6 +208,7 @@ public void testFilterApproximate() throws Exception {
         QueryResult query = getFunnelQueryExecutor().query(PROJECT_NAME,
                 of(new FunnelStep("test0", Optional.of("teststr = 'test1'")), new FunnelStep("test1", Optional.of("teststr = 'test1'"))),
                 Optional.of("teststr"),
+                Optional.empty(),
                 LocalDate.ofEpochDay(0),
                 LocalDate.ofEpochDay(SCALE_FACTOR), Optional.empty(), UTC,
                 Optional.empty(), APPROXIMATE).getResult().join();
```

Comment:
It would be nice to have a few tests with segments.

Reply:
I have added 3 tests covering segment for ordered, default and approximate